qcengine/procedures/base.py

```diff
@@ -64,3 +65,4 @@ def list_available_procedures() -> Set[str]:
 
 register_procedure(GeometricProcedure())
 register_procedure(BernyProcedure())
+register_procedure(NWChemRelaxProcedure())
```

Comment:
shouldn't this be `NWChemDriverProcedure`, too?

Reply:
Agreed and changed